src/instructlab/configuration.py

```diff
@@ -146,6 +146,10 @@ class _chat(BaseModel):
         default=None,
         description="The maximum number of tokens that can be generated in the chat completion. Be aware that larger values use more memory.",
     )
+    temperature: float = Field(
+        default=0.7,
```

Comment:
Is 0.7 what we always use as a default somewhere? It seems a bit high to have as the default. Have we tested what the output is like with this setting? We should make sure that this will give better results than other values.

Reply:
@alinaryan @RobotSail am I reading the old code right. It seems if `greed_mode` was not set then `create_params['temperature']` is never set, which means the temperature is 1 which is the default here: https://platform.openai.com/docs/api-reference/chat/create#chat-create-temperature.

I know we had discussed what the default temperature value should be. It looks like it's `1.0` in vLLM as well (https://github.com/vllm-project/vllm/blob/main/vllm/sampling_params.py#L176) and `0.8` in llama-cpp (https://github.com/ggerganov/llama.cpp/blob/master/examples/main/README.md?plain=1#L170).

Maybe we just leave it at 1.0 then?

Reply:
Does 1.0 seem like it is too high? Let me experiment with chat and I'll post results on this PR or in slack

Reply:
@alimaredia and I spoke offline and agreed that the default should be set to 1.0, since that is the OpenAI default anyways. That default serves as a reinforcement to our default so it makes sense for them to be the same value.